test/distributed/test_c10d_nccl.py

```diff
@@ -4189,7 +4188,6 @@ def setUp(self):
             "TORCH_NCCL_ENABLE_TIMING"
         ] = "0"  # see 'timing_enabled' parametrized tests
         os.environ["TORCH_NCCL_TRACE_BUFFER_SIZE"] = "1000"
-        os.environ["TORCH_NCCL_DUMP_ON_TIMEOUT"] = "1"
```

Comment:
it's a bit concerning that we'd have to update any tests at all.

(except for one case: if the test used to rely on /tmp/ as the write location without setting the env itself, then the test had a bug and needs to be updated to set the write location)

otherwise, the change in the PR should not affect existing tests should it?

Reply:
This was me removing that flag. I've put back the flag - so existing tests are now unaffected.